templates/dispatch-event.timer.j2

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Timer to dispatch {{event}} event periodically
+Requires=ghro.{{event}}.service
+
+[Timer]
+Unit=ghro.{{event}}.service
+OnUnitInactiveSec={{interval}}m
+RandomizedDelaySec={{jitter}}m
```

Comment:
oh, ok you're using systemd to emit `juju-run action` events on the current agent -- eliminating the need for cron.  Nifty